skimage/draw/_random_shapes.py

```diff
@@ -254,12 +254,13 @@ def random_shapes(image_shape,
         ``multichannel`` is set to False.
     shape : {rectangle, circle, triangle, None} str, optional
         The name of the shape to generate or `None` to pick random ones.
-    intensity_range : {tuple of tuples of ints, tuple of ints}, optional
+    intensity_range : {tuple of tuples of uint8, tuple of uint8}, optional
```

Comment:
This argument doesn't have to be of type uint8.  Perhaps:

```
intensity_range : {tuple of tuples of int in [0, 255], tuple of int in [0, 255)}
```

The special casing around 255 or 0 doesn't feel good.  Maybe we can make this more explicit (i.e., as mentioned below, perhaps allow specific background color to be set by default).